apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/Operations.java

```diff
@@ -328,6 +341,21 @@ private void writeBackupDataManifests(
     }
   }
 
+  private void createAndSetPermissionForBackupDir(Table table, String backupDir) {
+    try {
+      final FileSystem fs = fs();
+      Path backupDirPath = new Path(table.location() + "/" + backupDir);
+      if (!fs.exists(backupDirPath)) {
+        // set permission of backupDir to be write-only for table owner
+        FsPermission fsPermission =
+            new FsPermission(FsAction.WRITE_EXECUTE, FsAction.READ_EXECUTE, FsAction.NONE);
```

Comment:
Why would group have read permission when owner does not?

Reply:
We don't want the owner to view the backup dir, but will give `hdfsarchivalmgr` the permission to read and write.